python-sdk/tests/test_constants.py

```diff
@@ -7,7 +7,7 @@ def test_supported_file_locations():
 
 
 def test_supported_file_types():
-    expected = {"csv", "json", "ndjson", "parquet"}
+    expected = {"csv", "json", "ndjson", "parquet", "xls,xlsx"}
```

Comment:
Let's split this into two lines. We designed to have a file extension mapping 1:1 a `FileType` - this could break other parts of the code.